config/workers.coffee

```diff
@@ -318,7 +318,7 @@ module.exports = (KONFIG, options, credentials) ->
         ]
 
     userproxies         :
-      group             : 'proxy'
+      group             : if KONFIG.environment is 'default' then 'default' else 'proxy'
```

Comment:
I wanted to avoid tunnel workers in the default configuration. They depend on Route53 right now. I thought they would break the health check. I can try again, though.